prow/tide/tide.go

```diff
@@ -194,12 +199,14 @@ func (c *Controller) Sync() error {
 				pool = append(pool, pr)
 			}
 		}
-		allPRs, err := c.search(ctx, q.AllPRs())
+		allPRs, err := c.search(ctx, q.AllPRsSince(c.lastSyncStartTime))
 		if err != nil {
 			return err
 		}
 		all = append(all, allPRs...)
 	}
+	// We were able to find all open PRs so update the last sync time on return.
+	defer func() { c.lastSyncStartTime = syncStartTime }()
```

Comment:
Why bother deferring this?

Reply:
Just in case `c.lastSyncStartTime` ends up being used elsewhere in the `Sync` function in a future PR.

Reply:
YAGNI. Let's make that change in the future!